<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminWin</class>
 <widget class="QWidget" name="adminWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>950</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>950</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="bg">
   <property name="geometry">
    <rect>
     <x>-2</x>
     <y>5</y>
     <width>961</width>
     <height>601</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/resource/img/admin.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCerrarSesion">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>30</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px solid;
background: #921B63;
color:white;
border-radius: 9px;
font-weight:bold;
font-size:11px;
</string>
   </property>
   <property name="text">
    <string>CERRAR SESIÃ“N</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEliminar">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>110</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #B7387A;
background: none;
color: #B7387A;
border-radius: 15px;
font-weight:bold;
font-size:9px;
</string>
   </property>
   <property name="text">
    <string>ELIMINAR</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>GESTIONAR ELECCIONES</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnElecciones">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(0,0,0,0.6);
border: none;
background:none;
text-align: left;</string>
   </property>
   <property name="text">
    <string>Elecciones</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnVotantes">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(0,0,0,0.6);
border: none;
background:none;
text-align: left;</string>
   </property>
   <property name="text">
    <string>Votantes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCandidatos">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(0,0,0,0.6);
border: none;
background:none;
text-align: left;</string>
   </property>
   <property name="text">
    <string>Candidatos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCrear">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>110</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #B7387A;
background: none;
color: #B7387A;
border-radius: 15px;
font-weight:bold;
font-size:9px;
</string>
   </property>
   <property name="text">
    <string>CREAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAprobar">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #B7387A;
background: none;
color: #B7387A;
border-radius: 15px;
font-weight:bold;
font-size:9px;
</string>
   </property>
   <property name="text">
    <string>APROBAR DOCUMENTOS</string>
   </property>
  </widget>
  <widget class="QTableView" name="tbl">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>711</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #921B63;
background: rgba(255,255,255,0.5);</string>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEliminarVotante">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>110</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #B7387A;
background: none;
color: #B7387A;
border-radius: 15px;
font-weight:bold;
font-size:9px;
</string>
   </property>
   <property name="text">
    <string>ELIMINAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEliminarCandidato">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>110</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #B7387A;
background: none;
color: #B7387A;
border-radius: 15px;
font-weight:bold;
font-size:9px;
</string>
   </property>
   <property name="text">
    <string>ELIMINAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPropuesta">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #B7387A;
background: none;
color: #B7387A;
border-radius: 15px;
font-weight:bold;
font-size:9px;
</string>
   </property>
   <property name="text">
    <string>VER PROPUESTA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCambiarFase">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>110</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #B7387A;
background: none;
color: #B7387A;
border-radius: 15px;
font-weight:bold;
font-size:9px;
</string>
   </property>
   <property name="text">
    <string>CAMBIAR FASE</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
